Intersect.Editor/Core/Graphics.cs

```diff
@@ -1453,6 +1480,131 @@ bool alternate
+                yoffset -= CurrentView.Top;
+            }
+
+            if (gridX == 0 && gridY == 0)
+            {
+                if ((!HideTilePreview || Globals.Dragging) && !screenShotting)
+                {
+                    tmpMap = TilePreviewStruct;
+                }
+            }
+
+            if (tmpMap == null)
+            {
+                return;
+            }
+
+            for (var y = y1; y < y2; y++)
+            {
+                for (var x = x1; x < x2; x++)
+                {
+                    var tmpEvent = tmpMap.FindEventAt(x, y);
+                    if (tmpEvent == null)
+                    {
+                        continue;
+                    }
+
+                    if (tmpEvent.Pages[0].Graphic == null)
+                    {
+                        continue;
+                    }
+
+                    var tmpGraphic = tmpEvent.Pages[0].Graphic;
+                    if (TextUtils.IsNone(tmpGraphic.Filename))
+                    {
+                        continue;
+                    }
+
+                    Texture2D eventTex = null;
+                    var destinationX = x * Options.TileWidth + xoffset;
+                    var destinationY = y * Options.TileHeight + yoffset;
+                    var sourceX = 0;
+                    var sourceY = 0;
+                    var width = 0;
+                    var height = 0;
+
+                    switch (tmpGraphic.Type)
+                    {
+                        case EventGraphicType.Sprite: //Sprite
+                            eventTex = GameContentManager.GetTexture(
+                                GameContentManager.TextureType.Entity, tmpGraphic.Filename
+                            );
+
+                            sourceX = (int)tmpGraphic.X * (eventTex.Width / Options.Instance.Sprites.NormalFrames);
+                            sourceY = (int)tmpGraphic.Y * (eventTex.Height / Options.Instance.Sprites.Directions);
+                            width = (eventTex.Width / Options.Instance.Sprites.NormalFrames);
+                            height = (eventTex.Height / Options.Instance.Sprites.Directions);
+
+                            break;
+                        case EventGraphicType.Tileset: //Tile
+                            eventTex = GameContentManager.GetTexture(
+                                GameContentManager.TextureType.Tileset, tmpGraphic.Filename
+                            );
+
+                            sourceX = (int)tmpGraphic.X * Options.TileWidth;
+                            sourceY = (int)tmpGraphic.Y * Options.TileHeight;
+                            width = (tmpGraphic.Width + 1) * Options.TileWidth;
+                            height = (tmpGraphic.Height + 1) * Options.TileHeight;
+
+                            break;
+                    }
+
+                    if (height > Options.TileHeight)
+                    {
+                        destinationY -= Options.TileHeight;
```

Comment:
Change to the following so the rendering matches the client.
`destinationY -= (height - Options.TileHeight);`

Reply:
My bad, I thought I mimicked it but I missed that one. :)